spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/data/vectorized/ColumnarBatchReader.java

```diff
@@ -170,6 +173,7 @@ Pair<int[], Integer> posDelRowIdMapping() {
      * @return the mapping array and the new num of rows in a batch, null if no row is deleted
      */
     Pair<int[], Integer> buildPosDelRowIdMapping(PositionDeleteIndex deletedRowPositions) {
+      LOG.debug("Building row id mapping from positional deletes");
```

Comment:
Nit: is the log necessary? There would be a lot of messages in a real use case when debug level is on. I doubt how useful it is.

Reply:
I'm fine with removing the debug logging for the method calls in `ColumnBatchReader`. I'll do so.

Reply:
Removed.